warehouse/accounts/views.py

```diff
@@ -53,6 +53,7 @@ def failed_logins(exc, request):
 @view_config(
     route_name="accounts.profile",
     renderer="accounts/profile.html",
+    uses_session=True,
```

Comment:
Ok, if you merge master into this branch you can do this now in a template:

```
{% csi "/path/to/some/url/" %}
Content That will Render before the include has happened, or if JS is disabled for some reason.
{% endcsi %}
```

The work that's left to do in this PR then is to remove the stuff that touches the current logged in user on the user profile page, and replace it with the client side include (csi) block that has the "default" message on it and add a new URL that just renders the bit that you would like replaced and would be user dependent. You'll need to handle both the logged in and logged out case in that included URL, because it will be included unconditionally (unless the end user doesn't have JS or something) and it will completely replace whatever you put inside the ``{% csi %}{% endcsi %}`` blocks.

You can see an example of such a view at https://github.com/pypa/warehouse/blob/master/warehouse/views.py#L287-L293 which really just exists to render the template (thus why it's body is completely empty besides returning an empty dict). If you need help getting that setup I can help with that though.